angularjs/src/lib/components/cardnumber/cardnumber.component.ts

```diff
@@ -14,7 +14,7 @@ export class CardNumberCtrl implements ng.IComponentController {
 export class CardNumberComponent implements ng.IComponentOptions {
   public controller = CardNumberCtrl;
   public template = `
-  <div class="md-card-number">
+  <div class="md-ng1-card-number">
```

Comment:
No need to change this since it wasn't modified in the scss/ng/card-number.scss.